### ISSUE TITLE
How to upgrade boa to 4.0.0 

### ISSUE THREAD

**Issue by Corvalan** [omega8cc/boa#1325]
I just performed an upgrade of the stable version but it is still v. 3.2.2.  ?

**Comment by omega8cc** [comment]
It's still in HEAD, almost released, so either wait another 12-24 hours or upgrade to HEAD -- we run it in production already.

**Comment by cvining** [comment]
I also just tried `barracuda up-stable system` but the result was still v. 3.2.2

Is 4.0.0 stable yet? 

Thanks for all the fish!!

**Comment by yazzou** [comment]
Hello 
The same for me. I am still using an old drupal 7.56 version installed with my 3.2.2 version and can't upgrade (now for long time; many months)

**Comment by omega8cc** [comment]
4.0.0 official release has been delayed as we have explained before on Gitter, so you should simply upgrade to HEAD which is actually the new stable, as explained in the changelog. We expect to be able to release new official stable any day now, since the recent flood of core critical and regressions releases stabilized.

**Comment by macmladen** [comment]
If we use HEAD now, will we be able to do change to **stable** once it is released?

**Comment by omega8cc** [comment]
@macmladen Yes, easily, because stable will be actually head, just tagged as stable at some point, very soon. You can always run upgrade to stable after upgrade to head. That is how it's supposed to work.

**Comment by yazzou** [comment]
Hello 
Any news for the date of the v4.0.0 official stable release ?

**Comment by omega8cc** [comment]
We will skip 4.0.0 and release as 4.0.1 due to too many changes already introduced in the meantime. Can be any moment at this stage, once we have a green light. Remember that head is actually barracuda stable, just switching to rolling release policy and naming conventions are tricky to handle without leaving behind those running older releases.
